packages/composite-checkout/src/components/checkout.js

```diff
@@ -96,32 +90,48 @@ const Container = styled.div`
 	}
 `;
 
-const Column = styled.div`
+const MainContent = styled.div`
 	background: ${props => props.theme.colors.surface};
-	padding: 16px;
 	width: 100%;
 	box-sizing: border-box;
+
 	@media ( ${props => props.theme.breakpoints.tabletUp} ) {
 		border: 1px solid ${props => props.theme.colors.borderColorLight};
-		margin-top: 32px;
+		margin: 32px auto;
 		box-sizing: border-box;
-		padding: 24px;
+		max-width: 556px;
 	}
 `;
 
-const MainContent = styled( Column )`
-	@media ( ${props => props.theme.breakpoints.tabletUp} ) {
-		max-width: 532px;
-	}
-`;
+function OrderSummaryStep( { CheckoutHeader } ) {
+	const localize = useLocalize();
 
-const PageTitle = styled.h1`
-	margin: 0;
-	font-weight: normal;
-	font-size: 24px;
-	color: ${props => props.theme.colors.textColorDark};
-	padding-bottom: 24px;
-`;
+	return (
+		<CheckoutStep
+			isActive={ false }
+			isComplete={ true }
+			stepNumber={ 0 }
+			title={ localize( 'You are all set to checkout' ) }
+			stepSummary={ <OrderSummaryContent CheckoutHeader={ CheckoutHeader } /> }
+		/>
+	);
+}
+
+OrderSummaryContent.propTypes = {
+	CheckoutHeader: PropTypes.elementType,
+};
+
+function OrderSummaryContent( { CheckoutHeader } ) {
+	return (
+		<React.Fragment>
```

Comment:
I could be wrong, but I think we don't need this `Fragment` since there's just one element being returned.

Reply:
I just saw your other note about an alternate approach.